code/src/components/AddItemForm.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todo } from 'reducers/todo'
+
+export const AddItemForm = () => {
+  const [name, setName] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(todo.actions.addItem(name))
+    setName('') //clears the input field
```

Comment:
to make this even better you could check so that the name variable isn't empty before saving it as a todo? right now you could create empty todos.
```
const handleSubmit = (event) => {
  event.preventDefault()
  
  if (name) {
    dispatch(todo.actions.addItem(name))
    setName('') //clears the input field
  }
}
```